buildSrc/src/main/groovy/bio.terra.profile.java-spring-conventions.gradle

```diff
@@ -6,6 +6,5 @@ plugins {
     id 'org.springframework.boot'
 }
 
-// Spring Boot 3.2.3 pulls in opentelemetry-bom 1.31.0.
-// It must have version >= 1.34.1 for compatibility with terra-common-lib 1.0.9:
-ext['opentelemetry.version'] = '1.36.0'
+// This version should match the one in terra-common-lib
+// ext['opentelemetry.version'] = '1.42.1'
```

Comment:
TCL versions >= 1.1.25 no longer use `ext['opentelemetry.version']`; they rely on Spring and otel BOMs to specify the proper versions. Before this PR merges I will delete the commented-out code.